### PR TITLE
RavenDB-22512 Change build number extraction

### PR DESCRIPTION
**Pull request by hiddenshadow21**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -90,27 +90,40 @@ jobs:
...
-              ;;
-          esac
-          filePath="RavenDB/Server/Raven.Server.dll"
-          tar -xvf $tarName $filePath
-          sudo apt install -y libimage-exiftool-perl
-          number=$(exiftool $filePath | grep 'File Version Number' | cut -d. -f 4)
-          echo "BUILD_NUMBER=$number" >> $GITHUB_ENV
+          version=$(echo ${{ inputs.version }} | cut -d. -f1,2 | tr -d .)
+          diff=$((version - 61))
+          if [[ $diff -ge 0 ]]; then
```
Comment (gregolsky) [inline .github/workflows/build.yml]: let's compare these number lexicographically
```
gregolsky@Grzegorz-Lachowski-LAPTOP:/mnt/d/work$ [[ '6.1' > '6.2' ]]; echo $?
1
gregolsky@Grzegorz-Lachowski-LAPTOP:/mnt/d/work$ [[ '6.3' > '6.2' ]]; echo $?
0
gregolsky@Grzegorz-Lachowski-LAPTOP:/mnt/d/work$ [[ '7.3' > '6.2' ]]; echo $?
0
gregolsky@Grzegorz-Lachowski-LAPTOP:/mnt/d/work$ [[ '5.3' > '6.2' ]]; echo $?
1
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -90,27 +90,40 @@ jobs:
...
+            filename="RavenDB-${{ inputs.version }}-linux-x64.tar.bz2"
+            wget -P x64 "https://daily-builds.s3.amazonaws.com/${filename}"
+            filePath="RavenDB/Server/Raven.Server"
+            tar -xvf x64/$filename $filePath
+            number=$(./$filePath -- info | grep "AssemblyVersion" | cut -d" " -f2)
```
Comment (gregolsky) [inline .github/workflows/build.yml]:
```suggestion
            number=$(./$filePath --info | grep "AssemblyVersion" | cut -d" " -f2)

```

Comment (gregolsky) [inline .github/workflows/build.yml]: can you try launching the version you already have on the disk with `qemu`

Comment (gregolsky) [reply .github/workflows/build.yml]: "QEMU can run aarch64 binaries on x64 host: qemu-aarch64 ./program"

Comment (hiddenshadow21) [reply .github/workflows/build.yml]: I've used docker to run it.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -106,11 +110,40 @@ jobs:
...
+              source ../set-ubuntu-${{ matrix.ubuntuName }}.sh
+              source ../set-raven-platform-${arch}.sh
+              export DEB_BUILD_ENV_IMAGE="ravendb-server-info-extractor_$DEB_ARCHITECTURE"
+              docker pull --platform $DOCKER_BUILDPLATFORM ubuntu:$DISTRO_VERSION
+              docker build \
+                --platform $DOCKER_BUILDPLATFORM \
+                --build-arg "DISTRO_VERSION=$DISTRO_VERSION" \
+                --build-arg "QEMU_ARCH=$QEMU_ARCH" \
+                -t $DEB_BUILD_ENV_IMAGE \
+                -f ../../../../../../current/dockerfile .
```
Comment (gregolsky) [inline .github/workflows/build.yml]:
```suggestion
                -f ../../../../../../current/Dockerfile .

```

**Review by gregolsky — APPROVED** [review]

**Pull request merged**